views/scss/qti/_tooltip.scss

```diff
@@ -25,4 +25,11 @@
 
 .qti-item .tooltip{
     padding: 10px;
+}
+
+.tooltip-active {
+    position: static;
+    border-style: none;
+    padding: 0 0 0 0;
+    font-size: 1em;
```

Comment:
Is the `em` unit accurate here? Unless good reason (for instance size should be relative to parent), we should use `rem` instead.

Reply:
The `em` is accurate because in this case we want the to be size of the parent which should be the tetx size.